website/docs/migrations/11-12.md

```diff
@@ -218,3 +218,46 @@ const projectConfig = {
 	},
 };
 ```
+
+### Update ES Lint config
+
+**Migration time: ~10min.**
+
+Depending on the project current state you might need to account for time spent fixing errors and warnings produced by the new ES Lint settings so the migration time may vary.
+
+ES Lint configuration has been updated. To make sure everything works with the new version update the project node version to 20 or higher and update the ES Lint configuration. The steps are described below.
+
+1. Delete the `.eslintrc.js`  and `.eslintignore` files from the theme root.
+
+2. Add `eslint.config.mjs` file to the theme root with the following content:
+```js
```

Comment:
don't add the exact code here as if something changes we need to update this guide. Add a link where they can see the code and copy it

Reply:
If you are referring to the Eightshift Frontend Libs config [eslint.config.mjs](https://github.com/infinum/eightshift-frontend-libs/blob/main/eslint.config.mjs) file this example is different as it includes the ignore pattern which replaces the values from the `.eslintignore` deprecated file

Reply:
well if this needs to be added in the config we should add it. @goranalkovic-infinum please check if this needs to be added

Reply:
@iruzevic We should replace the files in the default setup (standard version), but it should also stay in the guide since you won't have those files on existing projects.